src/templates/index.hbs

```diff
@@ -367,25 +383,25 @@
               <br>
 
 
-          <div class="row">
-             <div class="col-md-offset-1 col-md-3">
-               <img src="/assets/img/andreluisgomes.jpeg" class="img-responsive img-circle" />
+          <div class="row staff">
```

Comment:
add section class="staff" to semantic.

Reply:
Removed the `staff` CSS class from the `row` elements and renamed the enclosing `section` to `staff` from the former name: `weekly`. Also updated the CSS rules accordingly.